src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs

```diff
@@ -31,31 +33,67 @@ public void GivenThatCapabilitiesHaveBeenProvidedForTheSolution()
         [Then(@"the Capability Name")]
         public void ThenTheCapabilityName()
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityNames().Should().HaveCount(solutionCaps.Count());
         }
         
         [Then(@"Capability Version")]
         public void ThenCapabilityVersion()
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityVersions().Should().HaveCount(solutionCaps.Count());
         }
         
         [Then(@"Capability Description")]
         public void ThenCapabilityDescription()
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityDescriptions().Should().HaveCount(solutionCaps.Count());
```

Comment:
it looks like you've took this step out of the feature file, so do you need this step? And are we covering all the ACs without it?

Reply:
This is still in the tests